Diffsims simulate atomic diffraction pattern.ipynb

```diff
@@ -0,0 +1,1985 @@
+{
```

Comment:
Please can we remove this notebook from here and create a pyxem/diffsims-demos repository for tutorials like this.